specifyweb/frontend/js_src/lib/utils/parser/definitions.ts

```diff
@@ -168,7 +168,9 @@ export const parsers = f.store(
     'java.sql.Timestamp': {
       type: 'date',
       minLength: fullDateFormat().length,
-      // maxLength: fullDateFormat().length,
+      maxLength: fullDateFormat().length,
```

Comment:
I think we needed to remove the `maxLength` constraint as part of this pull request - because it was preventing entering values like `today + 2 weeks` in the query builder. Can you remove this line?